contracts/facets/ViewStateFacet.sol

```diff
@@ -202,10 +203,29 @@ contract ViewStateFacet {
             memory statsList = new ArchaeologistStatistics[](addresses.length);
 
         for (uint256 i = 0; i < addresses.length; i++) {
+            // Count the failures for the arch
+            uint256 failures = 0;
+
+            // Get arch sarco ids
+            bytes32[] memory sarcoIds = this.getArchaeologistSarcophagi(addresses[i]);
```

Comment:
Could we retrieve `s.archaeologistSarcophagi[archaeologist]` directly from appstorage and use a reference type instead of allocating a new `bytes32[]` in memory?

`bytes32[] storage sarcoIds = s.archaeologistSarcophagi[archaeologist]`

Then we might not even need that utility `getArchaeologistSarcophagi` unless it's being used client side

Reply:
Good idea